<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryEntry</class>
 <widget class="QWidget" name="InventoryEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#InventoryEntry {
	border: 1px solid;
	border-radius: 8px;
	background-color: #ffffff;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="itemType">
     <property name="text">
      <string>Item type</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="quantity">
     <property name="text">
      <string>Quantity</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="expiration">
     <property name="text">
      <string>Expires Date</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="infoBtn">
     <property name="text">
      <string>i</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="consumeBtn">
     <property name="text">
      <string>Consume</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="throwOutBtn">
     <property name="text">
      <string>Throw out</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="removeBtn">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 100, 100);</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
